docker-compose.yml

```diff
@@ -64,6 +64,22 @@ services:
       - MYSQL_PORT=3306
       - MYSQL_DATABASE=activitypub
 
+  backfill-ghost-explore-follows:
+    build: jobs/backfill-ghost-explore-follows
+    environment:
+      - MYSQL_USER=ghost
+      - MYSQL_PASSWORD=password
+      - MYSQL_HOST=mysql
+      - MYSQL_PORT=3306
+      - MYSQL_DATABASE=activitypub
+      - BATCH_SIZE=10
+      - MAX_CONCURRENT=5
+      - GHOST_EXPLORE_AP_ID=https://mastodon.social/users/testingshtuff
```

Comment:
Leftover testing?

Reply:
Kinda, I didn't want to use the real account when running locally - i think this should stay